packages/analytics/client/src/analytics_client/analytics_client.ts

```diff
@@ -39,6 +40,7 @@ import { TelemetryCounterType } from '../events';
 import { ShippersRegistry } from './shippers_registry';
 import { OptInConfigService } from './opt_in_config';
 import { ContextService } from './context_service';
+import { schemaToIoTs, validateSchema } from '../schema/validation';
```

Comment:
I tried importing this conditionally and via `await import` and it still loads it in the starting chunk of `core`... Could it be that the current packaging system doesn't understand how to break into chunks when the dynamic import occurs inside the package?

Reply:
```typescript
let potentiallyUndefinedAPI: typeof API | undefined;
...
import(path).then((api) => {
  potentiallyUndefinedAPI = api;
});
```